src/condor_tests/test_jobsets.py

```diff
@@ -0,0 +1,138 @@
+#!/usr/bin/env pytest
+
+import htcondor
+import logging
+import os
+from pathlib import Path
+import subprocess
+import time
+
+from ornithology import *
```

Comment:
Editorializing a bit here, but we should probably get away from doing this and import only the objects needed rather than filling up the namespace with everything in ornithology. Not worth rejecting the PR though.

Reply:
I'm just going to set this as resolved, nothing that needs major attention.